src/test/resources/conf/mongo.properties

```diff
@@ -0,0 +1,3 @@
+connectionHost=localhost
```

Comment:
This is added to the test properties, but I can't find any unit tests that use it?

Reply:
I've added a task to do this as its outside of the scope of this ticket #454